<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1604923904584' stop='1604923915602'>
    <name>Testando tela de Registro</name>
    <title>Testando tela de Registro</title>
    <test-cases>
        <test-case start='1604923904584' status='failed' stop='1604923905214'>
            <name>Deve navegar para Signup</name>
            <title>Deve navegar para Signup</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: No element found using locator: By(xpath, //h4[text()='Login'])</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(xpath, //h4[text()='Login'])
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/built/element.js:814:27
                    at ManagedPromise.invokeCallback_ (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at processTicksAndRejections (node:internal/process/task_queues:93:5)Error
                    at ElementArrayFinder.applyAction_ (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as getText] (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.&lt;computed> [as getText] (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/built/element.js:831:22)
                    at SignUpPage.tituloLogin (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/page-objects/signup.po.ts:16:27)
                    at Object.&lt;anonymous> (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:25:53)
                    at step (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:32:23)
                    at Object.next (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:13:53)
                    at fulfilled (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:4:58)
                    at processTicksAndRejections (node:internal/process/task_queues:93:5)
                From: Task: Run fit("Deve navegar para Signup") in control flow
                    at UserContext.&lt;anonymous> (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4172:9
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasminewd2/index.js:64:48
                    at ControlFlow.emit (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/events.js:62:21)
                    at ControlFlow.shutdown_ (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2674:10)
                    at /home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2599:53
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:23:5)
                    at addSpecsToSuite (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/home/local/CONDUCTOR/caio.maia/.nvm/versions/node/v15.0.1/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/e2e/src/specs/signup.e2e-spec.ts:6:1)
                    at Module._compile (node:internal/modules/cjs/loader:1083:30)
                    at Module.m._compile (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/node_modules/ts-node/src/index.ts:400:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (/home/local/CONDUCTOR/caio.maia/Documentos/Testes/protractor-test/alurapic/node_modules/ts-node/src/index.ts:403:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923905214' status='passed' stop='1604923915600'>
            <name>Deve mostrar Full name is required!</name>
            <title>Deve mostrar Full name is required!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1604923915600' status='pending' stop='1604923915600'>
            <name>Deve mostrar Email is required!</name>
            <title>Deve mostrar Email is required!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915600' status='pending' stop='1604923915600'>
            <name>Deve mostrar User name is required!</name>
            <title>Deve mostrar User name is required!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915600' status='pending' stop='1604923915600'>
            <name>Deve mostrar Password is required!</name>
            <title>Deve mostrar Password is required!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915600' status='pending' stop='1604923915600'>
            <name>Deve mostrar Invalid e-mail</name>
            <title>Deve mostrar Invalid e-mail</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915601' status='pending' stop='1604923915601'>
            <name>Deve mostrar Username already taken</name>
            <title>Deve mostrar Username already taken</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915601' status='pending' stop='1604923915601'>
            <name>Deve cadastrar usuario</name>
            <title>Deve cadastrar usuario</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1604923915601' status='pending' stop='1604923915601'>
            <name>Deve verificar se o usuario está cadastrado</name>
            <title>Deve verificar se o usuario está cadastrado</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>